apps/api/src/config/apiKey.ts

```
@@ -0,0 +1 @@
+export const API_KEY = process.env.API_KEY || "not_set";
```

What is this API Key for? + I'm not sure about setting a default value for an api key. For dev/testing, we could have it, but for prod, we definitely want the program to fail fast. Having a default value would make the program fail only when the api call is made. What do you think.